doc/firewalld_services.md

```diff
@@ -0,0 +1,83 @@
+
+
+## Open / Modify Firewall services
+
+A [firewalld service](http://www.firewalld.org/documentation/man-pages/firewalld.service.html) defines a set of ports, protocols and destination addresses simplyfing the process of allow them in a specific zone. 
+
+In `YaST`, the [CWMFirewallInterfaces](https://github.com/yast/yast-yast2/tree/master/library/network/src/modules/CWMFirewallInterfaces.rb) module provides a widget definition for manipulating the enablement of services in zones through a selection of interfaces (each interface belongs to a **ZONE**). The module has been adapted to work properly with the new `firewalld API`.
+
+![summaryofchannges](https://user-images.githubusercontent.com/7056681/35337660-b5700d4c-0113-11e8-829f-0d92cdfe97e3.png)
+
+Being the implementation something like:
+
+```ruby
+
+Yast.import "CWMFirewallInterfaces"
+
+# You can still use "service:" prepend although it is recommended to remove
+# it when adapting the module
+settings = { "services" => ["service:cluster"], "display_details" => true }
+CWMFirewallInterfaces.CreateOpenFirewallWidget(settings)
+```
+
+In most of the cases the only requirement will be to define the service in `firewalld` and probabily it will be already
+provided by the `firewalld` package. In summary the changes in code should look like:
+
+```ruby
+
+# Require the new firewalld library and drop any import of SuSEFirewall2
+require 'y2firewall/firewalld'
+
+  # In your Module.Read method replace SuSEFirewalld.Read by firewalld.read
+    firewalld.read
+  # # In your Module.Write method replace SuSEFirewalld.Write by firewalld.read
+    firewalld.write
+  
+  private
+  
+    # Of course also define the firewalld convenience method to obtain the firewalld singlenton instance
+    def firewalld
+      Y2Firewall::Firewalld.instance
+    end
+```
+
+### Modify service ports definition
+
+The services definition that cames with firewalld can be modified. By default all the services that came with firewalld are placed in `/usr/lib/firewalld/services` although if a service is modified then it is placed in `/etc/firewalld/services` allowing the admin to go bak to the original definition is needed.
+
+SuSEFirewallServices was used in the past for modifying the services definition, it has been also dropped and the services configuration have to be done through the new class `Y2Firewall::Firewalld::Service`. 
```

Comment:
I would say "service definitions" (as 'definition' is the noun here). What about something like (or something similar)?

*`SuSEFirewallServices` has been dropped, so services configuration should be done through the new `Y2Firewall::Firewalld::Service``class.*